cirq-core/cirq/ops/pauli_string.py

```diff
@@ -107,32 +107,56 @@
 
 @value.value_equality(approximate=True, manual_cls=True)
 class PauliString(raw_types.Operation, Generic[TKey]):
-    def __init__(
-        self,
-        *contents: 'cirq.PAULI_STRING_LIKE',
-        qubit_pauli_map: Optional[Dict[TKey, 'cirq.Pauli']] = None,
-        coefficient: 'cirq.TParamValComplex' = 1,
-    ):
-        """Initializes a new PauliString.
+    """Represents a multi-qubit pauli operator or pauli observable.
 
-        Examples:
-            >>> a, b, c = cirq.LineQubit.range(3)
+    `cirq.PauliString` represents a multi-qubit pauli operator, i.e.
+    a tensor product of single qubit (non identity) pauli operations,
+    each acting on a different qubit. For  example,
+
+    - X(0) * Y(1) * Z(2): Represents a pauli string which is a tensor product of
+                          `cirq.X(q0)`, `cirq.Y(q1)` and `cirq.Z(q2)`.
+
+    `cirq.PauliString` is often used to represent:
+    - Pauli operators: Can be inserted into circuits as multi qubit operations.
+    - Pauli observables: Can be measured using either `cirq.measure_single_paulistring`/
+                        `cirq.measure_paulistring_terms`; or using the observable
+                        measurement framework in `cirq.measure_observables`.
 
-            >>> print(cirq.PauliString([cirq.X(a), cirq.X(a)]))
-            I
+    PauliStrings can be constructed via various different ways, some examples are
+    given as follows:
 
-            >>> print(cirq.PauliString(-1, cirq.X(a), cirq.Y(b), cirq.Z(c)))
-            -X(0)*Y(1)*Z(2)
+        >>> a, b, c = cirq.LineQubit.range(3)
 
-            >>> print(cirq.PauliString({a: cirq.X}, [-2, 3, cirq.Y(a)]))
-            -6j*Z(0)
+        >>> print(cirq.PauliString([cirq.X(a), cirq.X(a)]))
+        I
```

Comment:
this one is super confusing.  should we explain that I is the empty PauliString somewhere?

Reply:
Done.